examples/category-results-materialize.html

```diff
@@ -3,7 +3,8 @@
   <head>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.8/css/materialize.min.css">
     <style>
-      ngc-omnibox-suggestions {
+      ngc-omnibox-suggestions,
+      ngc-omnibox-selections {
```

Comment:
Because non-standard divs are `display: inline` by default. I get around that on most of them by making them attribute directives, but these are transclusion slots so won't work here.